dali/util/nvml_wrap.cc

```diff
@@ -40,13 +40,6 @@ static nvmlReturn_t (*nvmlInternalDeviceGetCpuAffinity)(nvmlDevice_t device,
                                                         unsigned int cpuSetSize,
                                                         unsigned long* cpuSet);  // NOLINT(*)
 
-static nvmlReturn_t (*nvmlInternalDeviceGetBrand)(nvmlDevice_t device, nvmlBrandType_t *type);
-static nvmlReturn_t (*nvmlInternalDeviceGetCount_v2)(unsigned int *deviceCount);
-static nvmlReturn_t (*nvmlInternalDeviceGetHandleByIndex_v2)(unsigned int index,
-                                                             nvmlDevice_t* device);
-static nvmlReturn_t (*nvmlInternalDeviceGetArchitecture)(nvmlDevice_t device,
-                                                         nvmlDeviceArchitecture_t* arch);
```

Comment:
`nvmlDeviceArchitecture_t` is not defined in CUDA 10. Maybe you can create ifdef for CUDA 10 and say it is `void*`?